packages/fiori-elements-writer/test/__snapshots__/lrop.test.ts.snap

```diff
@@ -21828,33 +21815,6 @@ archive.zip
   },
   \\"sapux\\": true
```

Comment:
CAP project should not have `sapux` in the Fiori App package.json. It is an array in the CAP root package.json instead for Fiori Elements apps